Add MathExp-kpetrov2 (#1649)
Codecademy/docs: content/c-sharp/concepts/math-functions/terms/exp/exp.md

---
Title: '.Exp()'

> Description: 'Returns the result of raising e with a specific number.'

Reviewer: ```suggestion
Description: 'Returns the result of raising e to the power of a specific number.'
```

Subjects:
  - 'Computer Science'
  - 'Code Foundations'
Tags:
  - 'Arithmetic'
  - 'Numbers'
  - 'Methods'
CatalogContent:
  - 'learn-c-sharp'
  - 'paths/computer-science'
---

> The **`Math.Exp()`** method returns the value of raising _e_ the power of a given number.

Reviewer: ```suggestion
The **`Math.Exp()`** method returns the value of raising _e_ to the power of a given number.
```

> The **`Math.Exp()`** method returns the value of raising **`e`** with the number given as a **power**.

Reviewer: ```suggestion
The **`Math.Exp()`** method returns the value of raising _e_ to the power of a given number.
```

> **`e`** stands for Euler's number, the **Euler's number** is an irrational number, with an approximate constant value of **2.71828** , that provides a constant base to the **natural logarithm** or **`log`** and can be understood as a limit of a given progression.

Reviewer: ```suggestion
_e_ stands for Euler's number, an irrational number, with an approximate constant value of 2.71828. It provides a constant base to the natural logarithm or log and can be understood as a limit of a given progression.
```

$$
\sum\limits_{n=0}^{\infty} \frac{1}{n!} = 1 + \frac{1}{n+1} + \frac{1}{(n+1)(n+2)} + \frac{1}{(n+1)(n+2)(n+3)} + \cdots
$$

> Consequently, the **`Math.Exp()`** method is the inverse of the **`Math.Log()`** method.

Reviewer: ```suggestion
Consequently, the `Math.Exp()` method is the inverse of the `Math.Log()` method.
```
